index.html

```diff
@@ -1055,25 +1044,6 @@
       payload data of the <a>NDEF record</a> with an appropriate ECMAScript
       type, which depends on the <a>IANA media type</a>.
     </p>
-    <p>
-      A <dfn>valid NFCRecord</dfn> <var>record</var> is an instance of
-      <code><a>NFCRecord</a></code> which matches the following criteria:
-      <ul>
-        <li>
-          <var>record</var>.kind has values defined by
-          <code>NFCRecordType</code>, or <code>undefined</code>.
-        </li>
-        <li>
-          <var>record</var>.type is a <a>valid MIME type</a>, or
```

Comment:
Yes. Not all branches in algorithms require it being a valid MIME type, and the checks are done in the specific steps.